src/components/apply/formStyles.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { TextField, SelectField } from 'redux-form-material-ui';
+import MenuItem from 'material-ui/MenuItem';
+import { Field } from 'redux-form';
+import _ from 'lodash';
+
+import * as colors from 'src/helpers/colors';
+// Will be added in the future
+{/* import { required as requiredValidator } from './form_validators'; */}
+
+/**
+ * Applies all the provided validator functions to the input, returning the first rejection.
+ */
+const applyValidators = validators => (input, lastInput) => {
```

Comment:
Help me understand this method. Why are we applying all validators and then finding the first?